src/utils/workspace.ts

```diff
@@ -8,6 +8,19 @@ import * as vscode from 'vscode';
 import { IAzureQuickPickItem } from 'vscode-azureextensionui';
 import { IDeployContext } from '../commands/deploy/IDeployContext';
 import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+
+export async function selectWorkspaceFolder(placeHolder: string): Promise<string> {
```

Comment:
I think you can get rid of this now that you moved everything into getDeployNode.